paradise_papers_search/paradise_papers_search/constants.py

```diff
@@ -0,0 +1,9 @@
+from neomodel import db
+
+countries = db.cypher_query("MATCH (n) WHERE NOT n.countries CONTAINS ';' RETURN DISTINCT 'node' as entity, n.countries AS countries UNION ALL MATCH ()-[r]-() WHERE EXISTS(r.countries) RETURN DISTINCT 'relationship' AS entity, r.countries AS countries")[0]
+
+jurisdictions = db.cypher_query("MATCH (n) WHERE EXISTS(n.jurisdiction) RETURN DISTINCT 'node' as entity, n.jurisdiction AS jurisdiction UNION ALL MATCH ()-[r]-() WHERE EXISTS(r.jurisdiction) RETURN DISTINCT 'relationship' AS entity, r.jurisdiction AS jurisdiction")[0]
```

Comment:
maybe here we could use something like `MATCH (n)  RETURN DISTINCT n.jurisdiction AS jurisdiction`?